prov/efa/src/efa_hmem.c

```diff
@@ -153,19 +165,32 @@ static int efa_hmem_support_status_update_neuron(struct efa_hmem_support_status
 	}
 
 	neuron_status->p2p_supported = true;
+	/* Eager and runting read protocols */
+	if (-ENODATA != fi_param_get(&rxr_prov, "inter_max_medium_message_size", &tmp_value)) {
+		EFA_WARN(FI_LOG_DOMAIN,
+				"The following environment variable was set FI_EFA_INTER_MAX_MEDIUM_MESSAGE_SIZE, "
+				"but EFA HMEM via Neuron API only supports eager and runting read protocols\n");
```

Comment:
This seems like a confusing print-out, maybe it is missing a colon, (`variable was set: FI_EFA...`) or should read more naturally like `The environment variable FI_EFA... was set, but ...`

Reply:
done